tools/autodeployment/server.js

```diff
@@ -7,6 +7,7 @@ const mergeStream = require('merge-stream');
 const fs = require('fs');
 
 const { buildStart, buildStop, handleStatus } = require('./info');
+const { prodStatus } = require('./slack_app');
```

Comment:
We use the same code for `prod` and `dev`, so you can name this `status` or `telescopeStatus`.